### PR TITLE
Update wording for card payments 101

### PR DESCRIPTION
**Pull request by ncjones**

### CODE REVIEW

File: imsv-docs-astro/src/content/docs/guides/core-concepts/card-payments-101.mdoc [hunk 1]
```diff
@@ -34,13 +34,17 @@ have not yet cleared appear to the cardholder as “pending” transactions.
...
 
 ## Settlement
 
-All cleared transactions need to be paid out to Mastercard so that merchants
-can receive payment. The process of paying out cleared transactions is referred
-to as Settlement. Settlements are batched daily or at a cadence that is
-appropriate based on the current transaction volume.
+All cleared payment transactions need to be paid to Mastercard by Immersve. The
+process of paying out cleared transactions is referred to as Settlement.
+Settlements are are conducted by Immersve at a cadence as instructed by
```
Comment (catluin) [inline imsv-docs-astro/src/content/docs/guides/core-concepts/card-payments-101.mdoc:48]:
```suggestion
Settlements are conducted by Immersve at a cadence as instructed by
```

File: imsv-docs-astro/src/content/docs/guides/core-concepts/card-payments-101.mdoc [hunk 2]
```diff
@@ -34,13 +34,17 @@ have not yet cleared appear to the cardholder as “pending” transactions.
...
+Mastercard. An Immersve partner organisation does not actively participate in,
+or have visibility of, the settlement process.
```
Comment (catluin) [inline imsv-docs-astro/src/content/docs/guides/core-concepts/card-payments-101.mdoc:50]:
```suggestion
Mastercard. An Immersve partner organisation does not actively participate in the settlement process or has visibility of it.
```
I'd rewrite it with less commas but Hemingway Editor complains about my version too so I'm not sure what's best here. Maybe we don't need to mention visibility and just say that partner org is not participating?

File: imsv-docs-astro/src/content/docs/guides/core-concepts/card-payments-101.mdoc [hunk 3]
```diff
@@ -34,13 +34,16 @@ have not yet cleared appear to the cardholder as “pending” transactions.
...
 ## Settlement
 
-All cleared transactions need to be paid out to Mastercard so that merchants
-can receive payment. The process of paying out cleared transactions is referred
-to as Settlement. Settlements are batched daily or at a cadence that is
-appropriate based on the current transaction volume.
+All cleared payment transactions need to be paid to Mastercard by Immersve. The
+process of paying out cleared transactions is referred to as Settlement.
+Settlements are conducted by Immersve at a cadence as instructed by
+Mastercard. An Immersve partner organisation does not actively participate in the settlement process nor have visibility of it.
```
Comment (ncjones) [inline imsv-docs-astro/src/content/docs/guides/core-concepts/card-payments-101.mdoc:49]:
```suggestion
Mastercard. An Immersve partner organisation does not actively participate in
the settlement process nor have visibility of it.
```

**Review by NicBathgate — APPROVED** [review]
Nice

**Review by braden-holgate — APPROVED** [review]
Such clean diffs! This is clearer, nice!

**Pull request merged**